WcaOnRails/app/controllers/api/v0/competitions_controller.rb

```diff
@@ -23,29 +23,14 @@ def show
   end
 
   def show_wcif
-    # This is all the associations we may need for the competition WCIF!
-    # Since registrations are ordered later, associations inclusion for them is done later
-    includes_associations = [
-      :delegates,
-      :organizers,
-      { competition_events: [rounds: :competition_event] },
-      { competition_venues: { venue_rooms: [schedule_activities: :child_activities] } },
-    ]
-    competition = competition_from_params(includes_associations)
+    competition = competition_from_params
     require_can_manage!(competition)
 
     render json: competition.to_wcif
   end
 
   def update_wcif
-    includes_associations = [{
-      competition_venues: {
-        venue_rooms: {
-          schedule_activities: [{ child_activities: [:holder] }, :holder],
-        },
-      },
-    }]
-    competition = competition_from_params(includes_associations)
+    competition = competition_from_params
```

Comment:
Now that you've removed these two calls to `competition_from_params(associations)`, can we simplify `def competition_from_params(associations = {})` in this file? I think there is no one left calling it with any associations.

Reply:
Sounds good to me!